shared/components/_link.scss

```diff
@@ -0,0 +1,40 @@
+// Base
+%nio-link {
+  color: $palette-primary;
```

Comment:
Do we want to define a $palette-link instead? In our theme, $palette-link is $palette-primary, but if we wanted to change it, might be nice to have it up one level in _palette. What do you think?